packages/unmock-core/src/__tests__/validator.test.ts

```diff
@@ -40,29 +44,28 @@ const response: Response = {
   description: "foo bar",
 };
 
-describe("Tests findStatePathInService", () => {
+describe("Tests spreadStateFromService", () => {
   it("with empty path", () => {
     const resp = response.content as any;
-    const spreadState = StateRequestValidator.spreadStateFromService(
-      resp["application/json"].schema.properties,
+    const spreadState = spreadStateFromService(
+      resp["application/json"].schema,
       {},
     );
     expect(spreadState).toEqual({}); // Empty state => empty spread state
   });
 
   it("with specific path", () => {
     const resp = response.content as any;
-    const spreadState = StateRequestValidator.spreadStateFromService(
-      resp["application/json"].schema.properties,
+    const spreadState = spreadStateFromService(
+      resp["application/json"].schema,
       { test: { id: "a" } },
     );
     expect(spreadState).toEqual({
       // Spreading from "test: { id : { ... " to also inlucde properties
-      test: {
-        properties: {
-          id: {
-            type: "integer",
-            format: "int64",
+      properties: {
+        test: {
+          properties: {
+            id: null, // Will be removed due to wrong type
```

Comment:
Sets the value as needed. `null` is treated as a mismatch and is alerted when using higher-level functions.
`spreadStateFromService` is only exported for testing purposes here.